cmd/dagger/session.go

```diff
@@ -92,7 +97,7 @@ func EngineSession(cmd *cobra.Command, args []string) error {
 		paramBytes = append(paramBytes, '\n')
 		go func() {
 			if _, err := os.Stdout.Write(paramBytes); err != nil {
-				panic(err)
+				fmt.Fprintln(cmd.ErrOrStderr(), err)
```

Comment:
We don't need to panic here, but I think we should exit the process if this fails; the client will never be able to find the port so may as well die 💀

Reply:
Good point - added a `l.Close()` like the others, which should unblock the `http.Serve` below - seems to be what we do elsewhere, too.